libs/designer-ui/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "noFallthroughCasesInSwitch": true,
     "resolveJsonModule": true
   },
-  "files": ["../../types/Swagger.d.ts"],
+  "files": [],
```

Comment:
Wasn't sure if I should remove this line entirely or leave `[]`. Let me know if I should remove the line.